backend/app/api/endpoints/investor.py

```
@@ -1,19 +1,70 @@
+            max_investment=None,  # TODO: Add investment range fields to model
+        )
+        investors_with_users.append(investor_data)
+
+    return investors_with_users
+
+
+@router.get("/{investor_id}", response_model=InvestorProfileRead)
+async def get_investor_details(
+    investor_id: uuid.UUID,
+    session: SessionDep,
+    current_user: uuid.UUID = Depends(get_current_user),
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)